### ISSUE TITLE
ImportError: No module named json

### ISSUE THREAD

**Issue by kira0204** [pypa/manylinux#193]
Hi there,
I am trying to make wheels for one project in the docker image `quay.io/pypa/manylinux1_x86_64` but the  error as mentioned in the issue title is being raised https://travis-ci.org/kira0204/v8py/jobs/376977121#L535.
Version of python: `cp27-cp27m`

**Comment by mayeut** [comment]
`python2` called by `depot_tools` uses the first `python2` found in `PATH` (c.f. [v8py/setup.py](https://github.com/kira0204/v8py/blob/aba2fe4d8c1bcafe9098a4dd1656d633b29c6708/setup.py)). Here that's the one from CentOS 5 and that's python 2.4.3

```
docker run -it --rm quay.io/pypa/manylinux1_x86_64:latest
[root@839c25ea10cd /]# which python2
/usr/bin/python2
[root@839c25ea10cd /]# python2 -V
Python 2.4.3
```
The `json` module is available since python 2.6
`depot_tools` requires python 2.7.

**Comment by kira0204** [comment]
Thanks, is there any workaround?
